libs/community/langchain_community/document_loaders/directory.py

```diff
@@ -197,3 +198,55 @@ def load(self) -> List[Document]:
             pbar.close()
 
         return docs
+
+    def lazy_load(self) -> Iterator[Document]:
```

Comment:
Great if we're upgrading to lazy_load() could you delete the load() implementation? (it will proxy to lazy_load automatically)

Reply:
Deleted the previous load_file function, and made load proxy to lazy_load